benchmark/infrastructure/BenchmarkMeasurementContainer.h

```diff
@@ -0,0 +1,205 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (March of 2023, schlegea@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <concepts>
+#include <vector>
+#include <memory>
+#include <utility>
+
+#include "util/Exception.h"
+#include "util/Timer.h"
+#include "util/json.h"
+#include "../benchmark/infrastructure/BenchmarkMetadata.h"
+#include "../benchmark/infrastructure/CopybaleUniquePtr.h"
+
+// All the classes, that hold the measurements of measured benchmarks.
+namespace BenchmarkMeasurementContainer{
```

Comment:
Shelved until further notice.

All manner of implementation would lead to bad practice and the good practice versions would need some rather hefty changes everywhere. Not sure, if it would be worth it, especially, because it's only a sonar cloud complain.